client/src/plugins/libparsec/index.ts

```diff
@@ -33,7 +33,10 @@ class ParsecProxy {
       // eslint-disable-next-line no-prototype-builtins
       if (result && result.hasOwnProperty('ok') && !(result as { ok: boolean }).ok) {
         const resultError = result as { ok: boolean; error: { tag: string; error: string } };
-        window.electronAPI.log('debug', `Error when calling ${name}: ${resultError.error.tag} (${resultError.error.error})`);
+        // electronAPI is not available right at the start
+        if (window.electronAPI && window.electronAPI.log) {
+          window.electronAPI.log('debug', `Error when calling ${name}: ${resultError.error.tag} (${resultError.error.error})`);
+        }
```

Comment:
In that case should not we fallback to a plain `console.debug` ?

Reply:
You're absolutely right! I updated the code.